x-pack/plugins/files/server/blob_storage_service/adapters/es/es.test.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { Readable } from 'stream';
+import { promisify } from 'util';
+import { ElasticsearchClient } from '@kbn/core-elasticsearch-server';
+import { loggingSystemMock } from '@kbn/core-logging-server-mocks';
+import { elasticsearchServiceMock } from '@kbn/core/server/mocks';
+import { Semaphore } from '@kbn/semaphore';
+import * as sinon from 'sinon';
```

Comment:
It is better to stick to the built-in `jest.spyOn` function rather than include `sinon`.

Reply:
ah, very nice, TIL